src/main/java/org/fundacion/jala/converter/core/ExtractText.java

```diff
@@ -28,6 +30,8 @@
     private String language;
```

Comment:
In this file the line 29 should be removed

Reply:
Removed.